docs/technical.rst

```diff
@@ -21,17 +21,28 @@ Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
 Development should always be possible running the last two ``LTS`` Node.js versions,
 see Node.js `release schedule table <https://github.com/nodejs/Release#release-schedule>`_.
 
-====================== =================================================
-Node.js 8              Supported
-Node.js 10             Supported
-====================== =================================================
+====================== ================= ===============================
+Node Version           Status            Latest Status Change
+====================== ================= ===============================
+Node.js 8              Supported         2018-11-01
+Node.js 10             Supported         2018-11-01
+====================== ================= ===============================
 
 Node.js Features
 ^^^^^^^^^^^^^^^^
 
-``TODO``
+Buffer vs safe-buffer
+"""""""""""""""""""""
+Old Node versions up to versions ``4`` and ``5`` allowed for some unsafe usage
+of the ``Buffer`` API which led to the development of a replacement 
+`safe-buffer <https://github.com/feross/safe-buffer>`_ library. We have now for
+some time dropped support for distibution on all affected Node versions (mainly
+``4``) and usage of ``safe-buffer`` is not needed any more.
 
-Notes on ``Buffer``, ``safe-buffer``, stuff like that.
+.. note::
```

Comment:
I think this note might not be needed. I can instead remove all of the `safe-buffer` usages.

Reply:
Will keep this until `safe-buffer` usage is removed.